lewis/adapters/stream.py

```diff
@@ -86,7 +84,44 @@ def handle_accept(self):
             StreamHandler(sock, self.target)
 
 
-class Cmd(object):
+class CanNotProcessException(Exception):
+    pass
```

Comment:
From my point of view "And I could just call process_request with garbage values anyway... so it would still need to throw an exception on no match." is the key argument here. This is how I would write the function if someone told me "Write a function that processes something and return the result". I would make it raise an exception, I'm just not entirely sure which one...